README.md

```diff
@@ -19,11 +19,20 @@ using Dates
 using TimeZones
```

Comment:
Replace the red cross in PJM/RT LMP to the check
![image](https://github.com/user-attachments/assets/c83c49b3-82d3-4369-b4c2-0e6448606ea0)